bundle/05_spleen_segmentation_lightning.ipynb

```diff
@@ -0,0 +1,1126 @@
+    "\n",
+    "This work is licensed under the Creative Commons Attribution-ShareAlike 4.0 International License. To view a copy of this license, visit http://creativecommons.org/licenses/by-sa/4.0/.\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1a18d5cd-6338-4b41-87fd-4e119723bfee",
+   "metadata": {},
+   "source": [
+    "Let's start by initialising a bundle directory structure:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "e00b5416-dfab-4043-9293-ec2acdf5842d",
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2023-10-16T14:44:10.513242253Z",
+     "start_time": "2023-10-16T14:44:02.546210817Z"
+    }
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "/usr/bin/tree\n",
+      "SpleenSegLightning\n",
+      "├── configs\n",
+      "│   ├── inference.json\n",
+      "│   └── metadata.json\n",
+      "├── docs\n",
+      "│   └── README.md\n",
+      "├── LICENSE\n",
+      "└── models\n",
+      "\n",
+      "3 directories, 4 files\n"
+     ]
+    }
+   ],
+   "source": [
+    "%%bash\n",
+    "\n",
+    "python -m monai.bundle init_bundle SpleenSegLightning\n",
+    "which tree && tree SpleenSegLightning || true"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5888c9bd-5022-40b5-9dec-84d9f737f868",
+   "metadata": {},
+   "source": [
+    "## Metadata\n",
+    "\n",
+    "We'll first replace the `metadata.json` file with our description of what the network will do:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "b29f053b-cf16-4ffc-bbe7-d9433fdfa872",
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2023-10-16T14:45:11.617630093Z",
+     "start_time": "2023-10-16T14:45:11.573340254Z"
+    }
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Overwriting SpleenSegLightning/configs/metadata.json\n"
+     ]
+    }
+   ],
+   "source": [
+    "%%writefile SpleenSegLightning/configs/metadata.json\n",
+    "\n",
+    "{\n",
+    "    \"version\": \"0.0.1\",\n",
+    "    \"changelog\": {\n",
+    "        \"0.0.1\": \"Initial version\"\n",
+    "    },\n",
+    "    \"monai_version\": \"1.2.0\",\n",
+    "    \"pytorch_version\": \"2.0.0\",\n",
+    "    \"numpy_version\": \"1.23.5\",\n",
+    "    \"optional_packages_version\": {},\n",
+    "    \"name\": \"SpleenSegLightning\",\n",
+    "    \"task\": \"3D Spleen segmentation network using MONAI and Pytorch Lightning\",\n",
+    "    \"description\": \"This is a demo network for segmentation of the spleen from 3D MRI images.\",\n",
+    "    \"authors\": \"Oeslle Lucena\",\n",
```

Comment:
I'd suggest "Your Name Here" for author and copyright just to keep it generic.

Reply:
Cool! Done!